aos_sal/Dockerfile

```diff
@@ -1,20 +1,22 @@
 # Dockerfile to execute a base machine with all set to test active optics
 # system software
 
-FROM lsstts/develop-env:c0010
+ARG IMAGE_TAG
```

Comment:
Can you please provide default value? develop-env:latest?

Reply:
The problem is that Tiago will never follow this (`latest`) in `develop-env`. You can see he always ask us to use `c0015` or `b045` or something like this~ I guess you should try to persuade him first. Actually, I also felt it is annoying that I never understand the meaning of `c0xx` or `b0xx`.